packages/OmegaPrint-Core.package/OPPrinter.class/README.md

```diff
@@ -0,0 +1,12 @@
+An OPPrinter is a code prettifier. Therefor it uses Ohm/S as a parser. The OPPrinter is a OhmSourceRewriter and uses OhmExplicitSendsSmalltalk as grammar.
```

Comment:
a**n** OhmSourceRewriter

Reply:
Therefore oder wir lassen es weg: It uses Ohm/S as a parser.

Reply:
man kann therefor auch ohne e schreiben auch wenn das Wort scheinbar sehr alt ist